src/vs/base/browser/ui/grid/grid.ts

```diff
@@ -576,6 +576,17 @@ export class Grid<T extends IView = IView> extends Disposable {
 		return this.gridview.getViewCachedVisibleSize(location);
 	}
 
+	setViewsVisible(visible: boolean, exclude?: T): void {
```

Comment:
This feels like a strange method to have as API on the grid. What is it doing now that can't be done from the outside? And why would `exclude` be optional?